functions.php

```diff
@@ -474,3 +474,16 @@ function() {
 		}
 	}
 );
+
+/**
+ * Adds the canonical link from the sidebar menu to the post head
+ */
+function add_meta_tags() {
+	global $post;
+	if ( isset( $post->p4_seo_canonical_url ) ) {
```

Comment:
This seems to return true also if `p4_seo_canonical_url` is an empty string (so for example if it was set in the backend at some point and then removed), in that case you end up with a link with an empty href

Reply:
Thank you, just resolved this.